accessibility-checker-engine/test/v2/checker/accessibility/rules/landmark_name_unique_ruleunit/example_2_pass.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!--
+   /******************************************************************************
+     Copyright:: 2020- IBM, Inc
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  *****************************************************************************/
+-->
+
+<html lang="en-US">
+
+<head>
+</head>
+
+<body id="body">
+	<!-- PASS Example 1: This is two navs in different parts of the page with different labels. Passes everything, clearly differentiated. -->
+	<!-- passes because one is: main/nav where the other is: main/aside/nav -->
+	<main>
+		<div id="leftnav" role="navigation">
+			<h2 id="leftnavheading">Institutional Links</h2>
+			<ul>
+				<li>...a list of links here ...</li>
+			</ul>
+		</div>
+		<aside>
+			<div id="rightnav" role="navigation">
+				<h2 id="rightnavheading">Related topics</h2>
+				<ul>
+					<li>...a list of links here ...</li>
+				</ul>
+			</div>
+		</aside>
+	</main>
+
+	<!-- passes because one is: main/nav where the other is: main/footer/nav -->
```

Comment:
Why are the rest of the tests commented out?

Reply:
Deleted it. These were some old commented test cases that we were looking at that can be removed now.